lib/cwmp.ts

```diff
@@ -688,6 +688,26 @@ async function nextRpc(sessionContext: SessionContext): Promise<void> {
         ]);
       }
 
+      break;
+    case "getParameterAttributes":
+      // Set channel in case params array is empty
+      sessionContext.channels[`task_${task._id}`] = 0;
+      for (const p of task.parameterNames) {
```

Comment:
"for (const p of task.parameterNames)"  should be "for (const p of task.parameterNames || [])"
otherwise cwmp will crash when new device connected.
Error message is: a.parameterNames is not iterable. 
I guess parameterNames is undefined at that time.